Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ocaml/opam:alpine-ocaml-4.14 as builder
+
+RUN opam repo add opam git+https://github.com/ocaml/opam-repository
+RUN opam install dune
+
+USER root
+RUN apk add --no-cache libev-dev gmp-dev pkgconfig libpq-dev libressl-dev
+
+USER opam
+
+COPY --chown=opam muhokama.opam /build/muhokama.opam
+
+RUN cd /build && opam install . --deps-only --with-test -y
+
+COPY --chown=opam . /build/
+
+RUN cd /build && opam exec -- dune build
+
+FROM alpine:3 as final
+
+RUN apk add --no-cache libev-dev gmp-dev libpq-dev libressl-dev
```

Comment:
I'm pretty dumb with Docker but why this line is mandatory? (because of line 7).

Reply:
It's a multi stage docker build.
Line 7 is ran for the container to build binary and those libs are needed by some libraries.

Line 21 is for the final container tat embed the static binary built in the first container.
Event with this static binary it seems from my tests run that those system lib are required.